libsolidity/parsing/Parser.cpp

```diff
@@ -1556,7 +1561,8 @@ ASTPointer<VariableDeclarationStatement> Parser::parseVariableDeclarationStateme
 						ASTPointer<TypeName>(),
 						name,
 						ASTPointer<Expression>(),
-						Visibility::Default
+						Visibility::Default,
+						nullptr
```

Comment:
With the above change request in the AST.h header file, this code change could be reverted.